ui/component/viewers/videoViewer/internal/videojs-events.jsx

```diff
@@ -86,7 +95,21 @@ const VideoJsEvents = ({
         userId,
         uri,
         this, // pass the player
-        bitrateAsBitsPerSecond
+        bitrateAsBitsPerSecond,
```

Comment:
Sticking to "always original" for non lives?
`doTrackingBuffered` and `sendAndResetWatchmanData` uses `BANDWIDTH` for both types, though.

Reply:
yea, not too important for the start event, we know if it's original vs transcoded based on other factors. And then on watchman buffering it should be correct.